### ISSUE TITLE
First Gothic game support

### ISSUE THREAD

**Issue by DragonSWDev** [Try/OpenGothic#133]
Project description states: "The goal of this project is to make a feature complete Gothic client compatible with the game and mods." but completion status is only about Gothic 2 Night of the Raven and I wasn't able to find any clear status about Gothic 1 support. I've found some information (mostly in issues) that Gothic 1 should work but some things are missing (like trading stated on README.md or weapons stats) but nothing more. I understand that this projects currently focuses on implementing Gothic 2 NoTR but perhaps some Gothic 1 status on README.md should be provided. What do you think about this? Are there any plans to implement missing G1 features after G2 NoTR will be done?

**Comment by Try** [comment]
Hi, @DragonSWDev !

This project officially support only gothic2, because I don't have enough capacity to test both games.
So, the plan is to complete Gothic2 support first, and only then move forward.

**Comment by DragonSWDev** [comment]
Thank you for answer. I decided to check how Gothic 1 works with this implementation. Aside from obvious things (like different interface, different learning system etc.) and some bugs it's working fine. I don't know if it's possible to finish game but it's playable to some extent. Looks like extending support for Gothic 1 in future won't be very hard.

**Comment by Try** [comment]
> some bugs it's working fine

It works until you pass the letter to Milten :D - there is no steppers in mages temple, what break the game.

> different learning system etc

This should just work - learning is handle by script (unless it calls to deprecated functions)

**Comment by DragonSWDev** [comment]
> It works until you pass the letter to Milten :D - there is no steppers in mages temple, what break the game.

Yeah, I noticed this :D I was able to move game further by going to sleep. Milten was back in front of temple and I was able to finish quest. Also for some reason Diego was stuck on bridge when he was going back to the Old Camp. Sleeping also fixed this. There are some other minor issues but I think there is no sense pointing them now before Gothic 1 work will start.

> This should just work - learning is handle by script (unless it calls to deprecated functions)

I was talking about weapon skills (yeah, my bad, I should describe it better). Gothic 2 has percentage value of each (one handed, two handed, bow etc.) weapon skill. Gothic 1 has 3 levels of skills for each weapon type. I don't know if engine supports Gothic 1 skills levels but interface shows percentage skill just like on Gothic 2.

Anyway thank you for clarification and for your work.
